calculator.c

```diff
@@ -0,0 +1,44 @@
+/*Sean Kee*/
```

Comment:
Beautiful implementation! My only suggestion would be to add the function of your program in the comment at the top. Besides that, great job!